packages/firestore/test/integration/util/events_accumulator.ts

```diff
@@ -77,6 +83,7 @@ export class EventsAccumulator<
   }
 
   assertNoAdditionalEvents(): Promise<void> {
+    this.rejectAdditionalEvents = true;
     return new Promise((resolve: (val: void) => void, reject) => {
       setTimeout(() => {
         if (this.events.length > 0) {
```

Comment:
I think this code is now unreachable, since storeEvent will throw rather than adding to `this.events`.

Perhaps we should remove it (but still do the setTimeout so that tests using assertNoAdditionalEvents() will wait long enough for local events to hit the storeEvent code)?  Or does the waiting for local events not actually work as-is?

Reply:
TBH, I'm not sure if this function actually did anything. But I'm not certain of that either. I found that *in my situation*, this method would not object (prior to me adding 'rejectAdditionalEvents' at any rate.) But I'm not sure if other situations *would* trigger it. In the end, I took the coward's way out and left it there, figuring, whatever it was doing before (if anything) it'll continue doing (if anything). :)

It *seems* like it should be useful, for instance, if there were *already* events in the queue prior to calling this.